pkg/apis/apps/v1alpha1/zz_generated.openapi.go

```diff
@@ -24,6 +22,7 @@ func schema_pkg_apis_apps_v1alpha1_APIManager(ref common.ReferenceCallback) comm
 		Schema: spec.Schema{
 			SchemaProps: spec.SchemaProps{
 				Description: "APIManager is the Schema for the apimanagers API",
+				Type:        []string{"object"},
```

Comment:
See comment https://github.com/3scale/3scale-operator/pull/308/files#r371898758
This specific change seems that is added because you must be using a different operator-sdk version or go version.
However, in this file there are legit changes so we must include some of them